packages/components/organisms/f-checkout/src/components/Checkout.vue

```diff
@@ -256,13 +256,17 @@ export default {
          *
          */
         async initialise () {
+            const initialLoadPromises = [this.loadAvailableFulfilment()];
+
             this.setAuthToken(this.authToken);
 
-            if (!this.isLoggedIn) {
+            if (this.isLoggedIn) {
+                initialLoadPromises.push(this.loadCheckout());
+            } else {
                 await this.loadBasket();
             }
 
-            await Promise.all([this.loadCheckout(), this.loadAvailableFulfilment()]);
+            await Promise.all(initialLoadPromises);
         },
```

Comment:
I personally don't like how this looks in the end. Any thoughts?

Reply:
I prefer Promise.all([this.loadCheckout(), this.loadAvailableFulfilment()]); as it's more readable, does not require to look for the definition of initialLoadPromises

Reply:
But we don't always call `loadCheckout`. That's the method we need to call only if we're authenticated.

Reply:
You could do the below instead?

```javascript
this.isLoggedIn
? Promise.all([this.loadCheckout(), this.loadAvailableFulfilment()])
: Promise.all([this.loadAvailableFulfilment()])

Reply:
I had pretty much that in my previous commit (https://github.com/justeat/fozzie-components/pull/642/commits/3abfc4bd41dbdb5c941d53f5eabfd44c5ce92583) but didn't really like it either. But I guess it's still better than what I have now.